.github/CODEOWNERS.txt

```diff
@@ -0,0 +1,4 @@
+# For details on CODEOWNERS file, see
+# https://help.github.com/en/articles/about-code-owners
+
+# Default owner of this repo
```

Comment:
I don't believe we are adding the original author as a code owner. According to the code owner link that you have in this doc, the code owner needs write permissions to this repo. I would be surprised if we are giving them write permissions to the repo.

What write permission would entail: https://help.github.com/en/github/setting-up-and-managing-organizations-and-teams/repository-permission-levels-for-an-organization#repository-access-for-each-permission-level

Reply:
My comment was based on thinking that we would be forking from their repo:
- in the case of forking, we could simply rebase from their original repo without giving them write-access to our fork
- as this would be an external contributor, I am not sure if we want to give them free rein (write-access) on what is written in the `dwave-examples` repo without someone internally having reviewed it first. Even if they are the original author

In the context of a repo transfer:
- I'm uncertain if we could do repo transfers as "To transfer a repository that you own to an organization, you must have permission to create a repository in the target organization." Perhaps we could try with a dummy account that is not a member of the `dwave-examples` organization?  https://help.github.com/en/github/administering-a-repository/transferring-a-repository#about-repository-transfers

Reply:
That's a good catch, we should verify it's possible to transfer a repo to dwave-examples without giving them write access to org.

Also, I didn't point this out, but obviously, to give someone a write access to a repo under our org, we should make sure that user is trustworthy.

Reply:
I added both options. Let me know if you'd like help testing the ownership transfer option.

Reply:
Yes, I'd like help testing out the ownership transfer option, @akoszegi. Since you're not a member of the `dwave-examples` organization, could you please try transferring a dummy repo to the organization?

Reply:
If it works, I would be happy allowing both options (as you have written @akoszegi). If problems occur, we can always update this boilerplate.

Reply:
I got this error when I tried. Is there a way around that?
![image](https://user-images.githubusercontent.com/15724694/78049486-f89e4300-732f-11ea-9804-0773de094839.png)

Reply:
Hi @akoszegi and @randomir . I don't see a way around it. The documentation (see above "about repository transfers" link) was specific that one of the prerequisites for a repo transfer was that the person needs permission to create a repo in the organization. 

Since "outside collaborators can never create repositories" and only members of the organization have the chance to create repos, I don't think the external repo transfer idea is possible. (see picture in step 5 of https://help.github.com/en/github/setting-up-and-managing-organizations-and-teams/restricting-repository-creation-in-your-organization)

**tl;dr**: external contributor would need to become a member. Owner would need to give members the power to create repos. Don't think this is a scale-able solution.

note: we could always give the original author write-permissions to our fork of their repo. I don't see any technical github issues there, only potential branding issues.

Reply:
@akoszegi: thanks for testing and confirming the issue

Reply:
Thanks for testing this, @akoszegi and @m3ller. Let's keep it simple for now and talk about forking only.

Personally, I don't see a problem in inviting external users to `dwave-examples` -- it's just an account for D-Wave/Ocean-relevant examples, not our main GitHub account. But doing that would surely require a broader discussion.

Reply:
We have to pay per member for `dwave-examples`, unfortunately. Hence, my not scale-able comment.

Reply:
I changed it to just include forking.